user_interaction/templates/user_interaction/home_users.html

```diff
@@ -0,0 +1,118 @@
+                                                <div class="">
+                                                    <div class=""
+                                                         style="background-size: contain; background-position: center;
+                                                             background-image: url('{{ group.icon.url }}');
+                                                             background-repeat: no-repeat; width: 3em; height: 3em;
+                                                             margin-left: -1em; margin-top: -0.5em;"
+                                                    ></div>
+                                                </div>
+                                            {% endif %}
+                                        </div>
+                                        <div>
+                                            {{ group.name }}
+                                        </div>
+                                    </div>
+                                    <ul class="list-group list-group-flush">
+                                        <li class="list-group-item">
+                                            <div class="row">
+                                                <div class="col">
+                                                    Control board
+                                                </div>
+                                                <a class="col btn btn-sm btn-outline-primary"
+                                                   href="{% url "committees:group_general" group_id=group.id %}"
+                                                >   Go to
+                                                </a>
+                                            </div>
+                                        </li>
+                                        {% for shortcut in group.shortcut_set.all%}
+                                            <li class="list-group-item">
+                                                <div class="row">
+                                                    <div class="col">
+                                                        {{ shortcut.name|capfirst }}
+                                                    </div>
+                                                    <a class="col btn btn-sm btn-outline-primary"
+                                                       href="{{ shortcut.url }}"
+                                                    >   Go to
+                                                    </a>
+                                                </div>
+                                            </li>
+                                        {% endfor %}
+                                    </ul>
+                                </div>
+                            {% endfor %}
+                        </div>
+                    {% endif %}
+                {% endif %}
+
+                <div>
```

Comment:
I suggest to leave the COVID-activity-registration-notice on here somewhere (just like it is mentioned on the not-logged-in homepage)